src/main/groovy/de/dkfz/eilslabs/batcheuphoria/jobs/Job.groovy

```diff
@@ -88,12 +93,16 @@ class Job<J extends Job> {
      */
     private de.dkfz.roddy.execution.jobs.JobResult runResult
 
+    private GenericJobInfo jobInfo
```

Comment:
Maybe have a list of info objects. Then we could have a history (if neccessary.)
Add in a comment for what this field is.